src/pages/shelves/list/index.jsx

```diff
@@ -66,15 +70,28 @@ const ShelvesList = props => {
     countUnit: '개',
   };
 
+  const isTotlaCountUnderShelvesLimit = () => {
```

Comment:
이 함수가 하는 일이 많은 것 같아요. 토스트는 `handleAddShelf`로 옮기는 게 낫지 않을까요?

그리고

```suggestion
  const isTotalCountUnderShelvesLimit = () => {
```